lib/husky_musher/app.py

```diff
@@ -127,21 +135,41 @@ def main():
         # If not in REDCap project, create new record
         user_data = {'record_id': register_net_id(net_id)}
 
-    # TODO -- generate a survey link for a particular day
-    # We are awaiting finalization of the REDCap project to know how
-    # daily attestations (repeating instruments) will be implemented.
+    # Because of REDCap's survey queue logic, we can point a participant to an
+    # upstream survey. If they've completed it, REDCap will automatically direct
+    # them to the next, uncompleted survey in the queue.
+    event = 'enrollment_arm_1'
+    instrument = 'eligibility_screening'
+    repeat_instance = None
+
+    # If all enrollment event instruments are complete, point participants
+    # to today's daily attestation instrument.
+    # If the participant has already completed the daily attestation,
+    # REDCap will prevent the participant from filling out the survey again.
     if is_complete('eligibility_screening', user_data) and \
         is_complete('consent_form', user_data) and \
         is_complete('enrollment_questionnaire', user_data):
-        return f"Congrats, {net_id}, you're already registered under record ID " \
-            f"{user_data['record_id']} and your eligibility " \
-            "screening, consent form, and enrollment questionnaires are complete!"
 
-    # Generate a link to the eligibility questionnaire, and then redirect.
-    # Because of REDCap's survey queue logic, we can point a participant to an
-    # upstream survey. If they've completed it, REDCap will automatically direct
-    # them to the next, uncompleted survey in the queue.
-    return redirect(generate_survey_link(user_data['record_id']))
+        event = 'encounter_arm_1'
+        instrument = 'daily_attestation'
```

Comment:
The group developing the redcap project seems pretty good about knowing to tell us about changing the variables at this point - that being said, since we're going to have to explicitly depend on event name and instrument name, it may be worth a proactive communication to let them know those properties also shouldn't be changed without giving us a heads up

Reply:
Good point! This is also true for our ETL process. Misja has been our REDCap guardian so I don't expect this to be a problem.